demos/http/http_demo_plaintext/demo_config.h

```diff
@@ -26,19 +26,23 @@
 /******* DO NOT CHANGE the following order ********/
 /**************************************************/
 
-/* Logging related header files are required to be included in the following order:
+/* Logging config definition and header files inclusion are required in the following order:
```

Comment:
Reword: _"Include logging header files and define logging macros in the following order."_

Reply:
Replace the line across all config files.